rs/boundary_node/certificate_issuance/certificate_issuer/src/work.rs

```diff
@@ -26,6 +26,10 @@ use crate::{
     TASK_DELAY_SEC, TASK_ERROR_DELAY_SEC,
 };
 
+fn truncated(s: &str, n: usize) -> String {
```

Comment:
Do we really need a 1-liner function that is used only once?

Reply:
by all means :) i let You decide